utils/README.md

```diff
@@ -0,0 +1,78 @@
+
+### Pixel based Metrics
+
+#### Intersection over Union (IoU)
+
+The IOU metric is a pixel based metric which measures overlap using the number of pixels common between the ground truth and the predictions divided by the total pixels across both.
+<p align="center">
+<img src="https://latex.codecogs.com/svg.latex?\Large&space;\textrm{IoU}=\frac{\textrm{ground truth} \cap \textrm{prediction}}{\textrm{ground truth} \cup \textrm{prediction}}" title="\Large IoU" class="center" />
+</p>
+
+#### Dice Similarity Coefficient  (dice)
+
+The dice metric scores model performance by measuring overlap between the ground truth and predictions divided by sum of pixels of both ground truth and predictions.
+<p align="center">
+<img src="https://latex.codecogs.com/svg.latex?\Large&space;\textrm{Dice}=2\cdot \frac{\textrm{ground truth} \cap \textrm{prediction}}{\textrm{ground truth} + \textrm{prediction}}" title="\Large Dice" class="center" />
+</p>
+
+>**Note :** IoU and Dice metrics weigh factors differently, however both metrics are positively correlated. This means if model A is better than B then this is captured similarly in both metrics.
+
+#### Precision and Recall
+
+By plotting a confusion matrix which indicates ground-truth and predicted classes with number of pixels classified in each class, [precision](https://scikit-learn.org/stable/modules/generated/sklearn.metrics.precision_score.html#sklearn.metrics.precision_score) and [recall](https://scikit-learn.org/stable/modules/generated/sklearn.metrics.recall_score.html#sklearn.metrics.recall_score) is easily computed.
+<p align="center">
+<img src="https://latex.codecogs.com/svg.latex?\Large&space;\textrm{precision}=\frac{\textrm{true positives} }{\textrm{true positives} + \textrm{false positives}}" title="\Large Precision" class="center" />
+</p>
+
+<p align="center">
+<img src="https://latex.codecogs.com/svg.latex?\Large&space;\textrm{recall}=\frac{\textrm{true positives}}{\textrm{true positives} + \textrm{false negatives}}" title="\Large Recall" class="center" />
+</p>
+
+<!-- classes = A and B evaluating for class A -->
+Where,
+- True Positives (TP) = pixels correctly classified as class A
+- False Positives (FP) = pixels incorrectly classified as class A
+- False Negatives (FN) = class A pixels incorrectly classified as class B
+- True Negatives (TN) = pixels correctly classified as class B
+
+#### Matthews Correlation Coefficient (MCC)
+
+The MCC metric takes all four confusion matrix categories (TP, FP, FN and TN) into account which in turn provides a more reliable score. This [article](https://bmcgenomics.biomedcentral.com/articles/10.1186/s12864-019-6413-7) compares MCC to other widely used metrics. This metric is insusceptible to the imbalanced dataset factor. Also [Scikit-Learn](https://scikit-learn.org/stable/modules/generated/sklearn.metrics.matthews_corrcoef.html) defines this metric is simpler terms.
```

Comment:
I know the word `insusceptible`exists, but I would rather say `not sensitive to` or even `not susceptible to` .  Could the *imbalanced dataset factor* be replaced by *class imbalance* ?